pkg/clusterconfig/clusterconfig.go

```diff
@@ -79,6 +85,39 @@ func GetConfig() (*restclient.Config, error) {
 	return nil, fmt.Errorf("could not locate a kubeconfig")
 }
 
+func getInstallConfig(client *clientcore.CoreV1Client) (*installer.InstallConfig, error) {
+	clusterConfig, err := client.ConfigMaps(kubeSystemNamespace).Get(clusterConfigName, metav1.GetOptions{})
+	if err != nil {
+		logrus.Errorf("%s/%s was not found or is otherwise inaccessible", kubeSystemNamespace, clusterConfigName)
```

Comment:
It almost always a bad practice to log and return the error. It's up to the caller to decide if this worth to be logged.

Reply:
updated in next push